packages/python-editor/tsconfig.json

```diff
@@ -4,10 +4,5 @@
     "outDir": "lib",
     "rootDir": "src"
   },
-  "include": ["src/*"],
-  "references": [
-    {
-      "path": "../application"
-    }
-  ]
```

Comment:
is it ok to remove the reference to application here? I believe the `python-editor`  still requires application ?

Reply:
nope it doesn't, it's not in package.json at all, I'm not sure why this was here at all, might have been an overzealous copy during my --watch support work

Reply:
good, I guess I got tricked by @jupyterlab/application